src/test/java/jenkins/plugins/logstash/LogstashConfigurationMigrationTest.java

```diff
@@ -35,7 +35,7 @@
 
 @RunWith(PowerMockRunner.class)
 @PrepareForTest({ LogstashInstallation.class, Descriptor.class })
-@PowerMockIgnore({"javax.crypto.*"})
+@PowerMockIgnore({"com.sun.org.apache.xerces.*", "javax.crypto.*", "javax.xml.*", "org.xml.*"})
```

Comment:
https://wiki.jenkins.io/display/JENKINS/Java+11+developer+guidelines refers to this as the standard powermock fix